app/views/admin/users/_header.html.erb

```diff
@@ -0,0 +1,87 @@
+<% if @user.access_locked? %>
+  <div class="crayons-notice crayons-notice--danger py-2 pr-2 mb-3 flex justify-between items-center">
+    <p class="flex-1 flex">
+      <%= crayons_icon_tag(:lock, title: "Access locked", class: "mr-2") %>
+      This user has locked access.
+    </p>
+    <%= link_to "Unlock access", unlock_access_admin_user_path(@user), method: :patch, class: "c-link c-link--block c-link--branded" %>
+  </div>
+<% end %>
+
+<section class="crayons-card flex gap-4 m:gap-6 mb-2 m:mb-3 p-3 s:p-4 m:p-7">
+  <img src="<%= @user.profile_image_url_for(length: 192) %>" width="96" height="96" alt="<%= @user.name %> profile picture" class="radius-full bg-base-10 hidden s:block">
+  <div class="grid gap-2 flex-1">
+    <div class="s:flex items-center">
+      <div class="flex flex-1 items-center">
+        <h1 class="crayons-title lh-tight">
+          <%= @user.name %>
+        </h1>
+        <%= render "status" %>
+      </div>
+      <div class="flex relative justify-between s:justify-end gap-2 my-2 s:my-0">
+        <%= link_to "Visit profile", @user.path, class: "c-cta", target: "_blank", rel: "noopener" %>
+        <div class="dropdown-trigger-container">
+          <button type="button" class="c-btn c-btn--icon-alone dropdown-trigger" id="options-dropdown-trigger" aria-haspopup="true" aria-expanded="false" aria-controls="options-dropdown">
+            <%= crayons_icon_tag("overflow-vertical", title: "Options") %>
+          </button>
+          <div class="crayons-dropdown right-0 left-0 s:left-auto" id="options-dropdown">
```

Comment:
**Some** of these buttons inside that dropdown need to trigger appropriate modals... Currently they do nothing :)

@nickytonline I'd appreciate your help here. These can be empty modals for now and I'll move cheese around later on and style appropriately.